pkg/encryption/elgamal/common.go

```diff
@@ -12,6 +14,11 @@ import (
 // H_STRING H is a random point on the elliptic curve that is unrelated to G.
 const H_STRING = "gPt25pi0eDphSiXWu0BIeIvyVATCtwhslTqfqvNhW2c"
 
+// GenerateKey generates a new ECDSA key pair.
+func GenerateKey() (*ecdsa.PrivateKey, error) {
```

Comment:
This is used for testing, but across multiple packages.

Reply:
Gotcha, any way to rename this or move this to some test utils file?

This is exposed to user as elgamal.GenerateKey(), which might be confusing (since it might suggest the generation of a el gamal Private Key or el gamal Public Key). Maybe just rename to GenerateEcdsaKey?